boards/riscv/it8xxx2_evb/it8xxx2_evb.dts

```diff
@@ -47,6 +47,14 @@
 &adc0 {
 	status = "okay";
 };
+/* voltage comparator for test */
```

Comment:
please split into commits: board changes should have their own commit.

Reply:
Done